plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/branch/BranchPresenter.java

```diff
@@ -361,16 +364,27 @@ protected void onSuccess(Branch result) {
 
                                              @Override
                                              protected void onFailure(Throwable exception) {
-                                                 final String errorMessage = (exception.getMessage() != null)
-                                                                             ? exception.getMessage()
-                                                                             : constant.branchCreateFailed();
-                                                 final GitOutputConsole console = gitOutputConsoleFactory.create(BRANCH_CREATE_COMMAND_NAME);
-                                                 console.printError(errorMessage);
-                                                 consolesPanelPresenter.addCommandOutput(appContext.getDevMachineId(), console);
-                                                 notificationManager.notify(constant.branchCreateFailed(), FAIL, true, project.getRootProject());
+                                                 if (getErrorCode(exception) == ErrorCodes.INIT_COMMIT_WAS_NOT_PERFORMED) {
+                                                     dialogFactory.createMessageDialog(constant.branchCreateNew(),
+                                                                                       constant.initCommitWasNotPerformed(),
+                                                                                       new ConfirmCallback() {
```

Comment:
No need to create an empty **ConfirmCallback** that do nothing.
You can pass null there. **ConfirmCallback** is nullable.